client/verta/tests/test_concurrency/test_concurrency.py

```diff
@@ -0,0 +1,33 @@
+import six
+import multiprocessing
+import pytest
+from .run_functions import log_observation, upload_artifact
+
+from operator import itemgetter
+from functools import partial
+
+
+class TestConcurrency:
+    @pytest.mark.skipif(six.PY2, reason="multiprocessing.Pool has issues in Python 2")
```

Comment:
@nhatsmrt Did you happen to check if these actually fail in Python 2? I am hoping that putting the parallel functions in a separate module would resolve that problem, but not sure

Reply:
Actually, I'm able to run the tests on Python with small modifications (https://stackoverflow.com/questions/7447284/how-to-troubleshoot-an-attributeerror-exit-in-multiproccesing-in-python/35991842)

See the latest commit, but basically the problem I was having is that Pool doesn't play well with "with" statement.